demos/src/scss/_demos.scss

```diff
@@ -42,19 +45,20 @@ h2 {
 	filter: progid:DXImageTransform.Microsoft.gradient(startColorstr=#{$ie-hex-str},endColorstr=#{$ie-hex-str});
 }
 .o-grid-row {
-	background: #bdcaff;
-	margin-top: 10px;
-	margin-bottom: 10px;
+	background: #f6e9d8;
+	margin-top: 1rem;
+	margin-bottom: 1rem;
 
 	.o-grid-row {
-		background: #dc73ff;
+		background: #cec6b9;
```

Comment:
You can add it via the build service like you're requiring buttons

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.